strapi/strapi
# Remove didReceiveAPIRequest event

### What does it do?

It removes the didReceiveAPIRequest event

### Why is it needed?

Because the didReceiveAPIRequest event is collapsing Amplitude.

### How to test it?

Check in the apiTokens and users and permissions view if the event is being sent.

### Related issue(s)/PR(s)
